custom_components/unifi_insights/api/network/endpoints/devices.py

```
@@ -342,13 +342,13 @@ async def get_port_metrics(
         def _to_float(value: Any) -> float | None:
             try:
                 return float(value)
-            except (TypeError, ValueError):
+            except TypeError, ValueError:
```

Pylint (4.0.7)

[error] Parsing failed: 'multiple exception types must be parenthesized (unifi_insights.api.network.endpoints.devices, line 345)'

(E0001)